content/en/user-guide/integrations/aws-cli/index.md

```diff
@@ -37,71 +39,75 @@ $ export AWS_DEFAULT_REGION="us-east-1"
+## LocalStack AWS CLI (`awslocal`)
 
+`awslocal` serves as a thin wrapper and a substitute for the standard `aws` command, enabling you to run AWS CLI commands within the LocalStack environment without specifying the `--endpoint-url` parameter or a profile.
 
 ### Installation
 
-You can install the `awslocal` command via `pip`:
+Install the `awslocal` command using the following command:
 
 {{< command >}}
 $ pip install awscli-local[ver1]
 {{< / command >}}
 
-Note that the command above also installs the latest version of the underlying AWS CLI version 1 (`awscli`) package. Use this command if you prefer to manage your own version of `awscli` (e.g., `v1`/`v2`) and install the wrapper script only:
+{{< alert title="Note" >}}
+The above command installs the most recent version of the underlying AWS CLI version 1 (`awscli`) package. If you would rather manage your own `awscli` version (e.g., `v1` or `v2`) and only install the wrapper script, you can use the following command:
+
 {{< command >}}
 $ pip install awscli-local
 {{< / command >}}
+{{< /alert >}}
 
 {{< alert title="Note" >}}
 Automatic installation of AWS CLI version 2 is currently not supported yet (at the time of writing there is no official pypi package for `v2` available), but the `awslocal` technically also works with AWS CLI v2 (see [this section]({{< ref "#limitations" >}}) for more details).
 {{< /alert >}}
 
 ### Usage
 
-The `awslocal` command has the same usage as the `aws` command.
-For detailed usage, please refer to the man pages of `aws help`.
+The `awslocal` command shares identical usage with the standard `aws` command. For comprehensive usage instructions, refer to the manual pages by running `awslocal help`.
 
 {{< command >}}
 awslocal kinesis list-streams
 {{< / command >}}
 
-### Configurations
-
-You can use the following environment variables for configuration:
-
-| Variable | Description |
-| -------- | ----------- |
-| `LOCALSTACK_HOST` | Set the hostname for the localstack instance. Useful when you have localstack is bound to another interface (i.e. docker-machine). |
-| `USE_SSL` | Whether to use `https` endpoint URLs (required if LocalStack has been started with `USE_SSL=true` enabled). Defaults to `false`. |
-| `DEFAULT_REGION` | *Deprecated*. Set the default region. Overrides `AWS_DEFAULT_REGION` environment variable. |
-
-Verify the current configuration:
-
-{{< command >}}
-awslocal configure list
-{{< / command >}}
+### Configuration
 
+| Variable         | Description                                                                                                   |
+| ---------------- | ------------------------------------------------------------------------------------------------------------- |
+| `LOCALSTACK_HOST` | Specifies the hostname for the LocalStack instance, particularly useful when LocalStack is bound to a different network interface, such as a docker-machine. |
```

Comment:
I think those are a bit outdated - the new canonical way to configure the target endpoint is via `AWS_ENDPOINT_URL`, and the `LOCALSTACK_HOST`/`USE_SSL` configs are deprecated. More details here: https://github.com/localstack/awscli-local#configurations